http/jsonapi/request_test.go

```diff
@@ -29,6 +29,7 @@ func TestUnmarshall_attrStringSlice(t *testing.T) {
 				"dec1": json.RawMessage(`"9.9999999999999999999"`),
 				"dec2": json.RawMessage("9.9999999999999999999"),
 				"dec3": json.RawMessage("10"),
+				"mss":  json.RawMessage(`{"field": ["string1","string2"]}`),
```

Comment:
this checks the happy path, could we also get a test that is expected to trigger the failure case or does Go's json unmarshal do magic converstion of JS Number to string by just wrapping "" around it, i.e.,

```json
`{"field": ["string1",1234, 9.9]}`
```
as test case.

Reply:
added